core/src/main/res/values/strings.xml

```diff
@@ -774,4 +774,7 @@
     <!-- Widget settings -->
     <string name="widget_create_button">Create widget</string>
     <string name="widget_opacity">Opacity</string>
+
+    <string name="subscription_auto_download">Auto Download</string>
+    <string name="subscription_auto_delete">Auto Delete</string>
```

Comment:
I think it is better to use the existing `pref_automatic_download_title` and `pref_auto_delete_title` string. Otherwise, the translations might be different.